packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -25,6 +25,16 @@ private class PigeonApiImplementation: ExampleHostApi {
     }
     completion(.success(true))
   }
+
+  func sendMessageModernAsync(message: MessageData) async throws -> Bool {
```

Comment:
you can just overload the function name `func sendMessage(...)`

Reply:
I don't think that I understood.

It's generated Host API code. We could not overload functions in Dart even if we do in Swift